addon-test-support/@ember/test-helpers/settled.ts

```diff
@@ -41,6 +43,23 @@ function pendingRequests() {
   return localRequestsPending + internalRequestsPending;
 }
 
+/**
+  @private
+  @returns {boolean} if there are any pending router transitions
+*/
+function hasPendingTransitions() {
+  const context = getContext();
```

Comment:
We are generally in the "liberally-let" camp (phrasing [from this blog post](https://madhatted.com/2016/1/25/let-it-be)). This should be `let`...